modules/sc-mesh-secure-deployment/src/nats/scripts/cli_settings_request.py

```diff
@@ -3,27 +3,80 @@
 import json
 import config
 
-
 async def main():
     # Connect to NATS!
     nc = await client.connect_nats()
-    cmd_dict = {"api_version": 1, "ssid": "test_mesh", "key": "1234567890",
-                "ap_mac": "00:11:22:33:44:55", "country": "FI", "frequency": "5220",
-                "frequency_mcc": "2412", "routing": "batman-adv", "priority": "long_range",
-                "ip": "192.168.1.2", "subnet": "255.255.255.0", "tx_power": "5", "mode": "mesh",
-                "role": f"{config.MODULE_ROLE}"}
+    cmd_dict = {
+        "api_version": 1,
+        "role": f"{config.MODULE_ROLE}",  # sleeve, drone, gcs
+        "radios": [
+            {
+                "radio_index": "0",
+                "ssid": "test_mesh2",
+                "key": "1234567890",
+                "ap_mac": "00:11:22:33:44:55",
+                "country": "US",  # all radios must have the same country
+                "frequency": "2412",
+                "frequency_mcc": "2412",  # multiradio not supporting
+                "routing": "batman-adv",
+                "priority": "long_range",
+                "ip": "10.20.15.3",
```

Comment:
Please remove all IPv4 or at least leave it empty to avoid the happened-several-times-already situations where we debug and debug and debug only to find out that several devices are using 10.20.15.3 etc :)

Reply:
I think coming clean-up with ipv6 can include all ipv4 remova